### PR TITLE
Updates Network Beaconing installation link

### LINKED ISSUE #4036: Update the public docs for Network Beaconing Identification package
## Description 

Since we've introduced Network Beaconing Identification as part of our integration packages, we aim to revise the public documentation page of this package. We'll eliminate the previous links that direct users to install the package through `detections-rules` repo and instead guide them to the updated integration documentation for the package.

## Related docs 
Old Links:
- https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md
- https://www.elastic.co/guide/en/security/current/network-beaconing-framework.html

New documentation:
- https://docs.elastic.co/integrations/beaconing

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves #4036.

Corresponding serverless PR: https://github.com/elastic/staging-serverless-security-docs/pull/217

Preview: [Network Beaconing](https://security-docs_4137.docs-preview.app.elstc.co/guide/en/security/master/network-beaconing-framework.html)

### CODE REVIEW

**Review by sodhikirti07 — APPROVED** [review]
Changes lgtm!

**Review by nastasha-solomon — APPROVED** [review]
Left one suggestion. Overall, everything looks good!

**Comment by sodhikirti07** [comment]
Could we add the below `Note` before `Feature details` section?

> Note: Integration [package](https://docs.elastic.co/integrations/beaconing) now provides a user-friendly experience to implement this detection package. If you wish to fine-tune any of the package components, you may install the package manually by following [these steps](https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md).

**Review by SourinPaul — APPROVED** [review]

**Comment by SourinPaul** [comment]
A quick reminder: we must also move this page from the `Technical Preview` section in 8.11.

I propose we host it under the 'Jobs in Advanced Analytics (UEBA) Elastic integrations' [https://security-docs_4137.docs-preview.app.elstc.co/guide/en/security/master/machine-learning.html#ml-integrations] section; by adding 'Network Beaconing' to the list of existing integration packages.

This also aligns with our plans to centralize detection use cases under the Advanced Entity Analytics menu within Elastic Security (ref: #3957, #4117 )

cc: @jmikell821

File: docs/experimental-features/beaconing-detection.asciidoc [hunk 1]
```diff
@@ -6,9 +6,11 @@ This feature provides an early warning system for command and control beaconing
 [discrete]
 === Deploy the package
 
-To deploy the network beaconing framework in your environment using the https://github.com/elastic/detection-rules/releases/tag/ML-Beaconing-20211216-1[release package], follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+To deploy the network beaconing framework in your environment, follow https://docs.elastic.co/integrations/beaconing#installation[these steps].
```
Comment (joepeeples) [inline docs/experimental-features/beaconing-detection.asciidoc:9]: This path could use the [attribute](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L127) for integrations docs:
```suggestion
To deploy the network beaconing framework in your environment, follow {integrations-docs}/beaconing#installation[these steps].
```

**Review by joepeeples — COMMENTED** [review]
One tiny nit (once CI gets back up)

**Comment by natasha-moore-elastic** [comment]
> Could we add the below `Note` before `Feature details` section?
> 
> > Note: Integration [package](https://docs.elastic.co/integrations/beaconing) now provides a user-friendly experience to implement this detection package. If you wish to fine-tune any of the package components, you may install the package manually by following [these steps](https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md).

Sure thing @sodhikirti07, I've added this in https://github.com/elastic/security-docs/pull/4137/commits/c2332b2a66b3b3110fb15af21d4c2208f741744c.

**Comment by natasha-moore-elastic** [comment]
@elasticmachine run elasticsearch-ci/docs

**Pull request merged**